RedHat-UX/red-hat-design-system
# docs: replace "Sample component" title

## What I did

1. Replaced "Sample component" titles on Element and Pattern pages with "Sample element" or "Sample pattern"

## Testing Instructions

1. Check [deploy preview](https://deploy-preview-1263--red-hat-design-system.netlify.app/)

## Notes to Reviewers
